### DISCUSSION TITLE
Allow to select the first image of a product gallery when using dynamic tags in the Flip Box

### DISCUSSION THREAD

**Discussion by tiago-sartor** [elementor/elementor#23350] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hello,

I wanted to create a hover effect for my products using the Flip Box widget. Recently I've watched this tutorial here https://www.youtube.com/watch?v=hkPgoU21efI  where the ACF plugin is used to create an alternative product image, which in turn is used to create the hover effect with the Flip Box widget in the product loop grid or carousel.
It would be a lot easier if we could just use the first image of the product gallery as the alternative image (as dynamic content), similarly as it is currently done for the featured image.

### Describe the solution you'd like

Make it possible to use the image in the product gallery as dynamic content to create hover effects with the Flip Box and Loop Carousel widgets.

### Describe alternatives you've considered

The alternative is to use third party plugins, but this is not optimal.

**Comment by narnyworld** [comment]
yup... I have been waiting for this feature for like 5 years... I am still using some "essential grid plugin", instead of the native loop/grid widget because of that 😥

**Comment by phkoon** [comment]
That should be set by default, as it it in most major Sales CMS out there.
Not having this even as a dynamic option is outrageous
